lsp-mode.el

```diff
@@ -1679,7 +1679,12 @@ This set of allowed chars is enough for hexifying local file paths.")
      ((and (not (file-directory-p file-name))
            (not (lsp--string-match-any ignored-files file-name))
            (memq event-type '(created deleted changed)))
-      (funcall callback event)))))
+      (funcall callback event))
+     ((and (not (file-directory-p file-name))
+           (not (lsp--string-match-any ignored-files file-name))
+           (memq event-type '(renamed)))
```

Comment:
can you put both memq checks on the first place in the `and`-s?

Reply:
Yes sure. Like so?